### PR TITLE
Add favicons for API Tools project

### PR DESCRIPTION
**Pull request by Arhell**
|    Q          |   A
|-------------- | ------
| Documentation | yes/no
| Bugfix        | yes/no
| BC Break      | yes/no
| New Feature   | yes
| RFC           | yes/no
| QA            | yes/no

### Description

add icons for https://api-tools.getlaminas.org/

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@froschdesign take a look please

File: public/images/favicon/api-tools/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "name": "API Tools",
+    "short_name": "API Tools",
+    "icons": [
+        {
+            "src": "/images/favicon/api-tools/android-chrome-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/images/favicon/api-tools/aandroid-chrome-512x512.png",
```
Comment (froschdesign) [inline public/images/favicon/api-tools/site.webmanifest:11]: The file name is wrong here.

File: public/images/favicon/api-tools/browserconfig.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square70x70logo src="/images/favicon/api-tools/amstile-70x70.png"/>
+            <square144x144logo src="/images/img/favicon/api-tools/amstile-144x144.png"/>
+            <square150x150logo src="/images/img/favicon/api-tools/amstile-150x150.png"/>
+            <square310x310logo src="/images/img/favicon/api-tools/amstile-310x310.png"/>
+            <wide310x150logo src="/images/img/favicon/api-tools/amstile-310x150.png"/>
```
Comment (froschdesign) [inline public/images/favicon/api-tools/browserconfig.xml:9]: The same here.

**Review by froschdesign — CHANGES_REQUESTED** [review]
I don't know exactly if we really need the configuration and webmanifest files, but the file names they contain are incorrect.

**Comment by Arhell** [comment]
@froschdesign done

**Comment by Arhell** [comment]
@froschdesign

**Review by froschdesign — APPROVED** [review]

**Comment by froschdesign** [comment]
@Arhell 
Thank you for your time and this contribution! 👍

**Pull request merged**